app/editor/src/features/login/UserInfo.tsx

```diff
@@ -26,7 +26,8 @@ export const UserInfo: React.FC<IUserInfoProps> = ({ children }) => {
       appStore.storeUserInfo();
       setInit(true);
     }
-  }, [app, appStore, keycloak.authenticated, init]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [appStore, keycloak.authenticated, init]);
```

Comment:
This worries me as we keep getting forced to remove dependencies.  The `app` is responsible for determining if the user info is loaded.  I wonder if removing this will result in some odd bugs with new users.

Reply:
Have you tested after clearing cache and local storage?

Reply:
Seems to work after clearing local storage and cache, just tried a few tests leaving app in as well and unfortunately couldn't escape the infinite loop